packages/main/src/ComboBox.js

```diff
@@ -712,6 +712,10 @@ class ComboBox extends UI5Element {
 		this._itemFocused = true;
 	}
 
+	get _placeholder() {
+		return this.placeholder || "";
```

Comment:
Isn't the default value already empty string, defined in the metadata defaultValue: ""

Reply:
> Would it work if you use

Once the placeholder is set to a different value and then removed, it does not fallback to the defaultValue instead it is set to null, therefore I made the getter to check and return the empty string.